packages/ComponentLibrary/src/components/index.ts

```diff
@@ -26,37 +23,34 @@ import SecondaryTabsMUI from './SecondaryTabs';
 import NotificationItemStates from './NotificationItemAllStates';
 import DrawerMUI from './Drawer';
 import IButton from './IconButton';
-import { BoxProps } from '@mui/material';
 
 export * from '../theme';
 
-// ApplyHOC is a higher order component that applies the theme to the component
+//  is a higher order component that applies the theme to the component
```

Comment:
Why is the reason to not apply the HOC with the styles ?

Reply:
The HOC wraps each component in its own ThemeProvider, but the ThemeProvider is already wrapping the entire App component, so, it is not need anymore